CHANGELOG.md

```diff
@@ -9,25 +9,24 @@
 
 ### Features
 
-* add always_use_jwt_access ([#714](https://www.github.com/googleapis/python-bigquery/issues/714)) ([92fbd4a](https://www.github.com/googleapis/python-bigquery/commit/92fbd4ade37e0be49dc278080ef73c83eafeea18))
```

Comment:
This use_jwt_access commit was later [reverted](https://github.com/googleapis/python-bigquery/pull/722).